DataFormats/MuonReco/interface/Muon.h

```diff
@@ -183,6 +183,43 @@ namespace reco {
 			   RPCHitAndTrackArbitration, GEMSegmentAndTrackArbitration, ME0SegmentAndTrackArbitration };
     
     ///
+    /// ====================== STANDARD SELECTORS ===========================
+    ///
+    enum Selector {
+      CutBasedIdLoose        = 1UL<< 0,  
+      CutBasedIdMedium       = 1UL<< 1,  
+      CutBasedIdMediumPrompt = 1UL<< 2,  // IP cuts
+      CutBasedIdTight        = 1UL<< 3,  
+      CutBasedIdHighPt       = 1UL<< 4,
+      PFIsoVeryLoose         = 1UL<< 5,  // reliso<0.40
+      PFIsoLoose             = 1UL<< 6,  // reliso<0.25
+      PFIsoMedium            = 1UL<< 7,  // reliso<0.20
+      PFIsoTight             = 1UL<< 8,  // reliso<0.15
+      PFIsoVeryTight         = 1UL<< 9,  // reliso<0.10
+      TkIsoLoose             = 1UL<<10,  // reliso<0.10
+      TkIsoTight             = 1UL<<11,  // reliso<0.05
+      SoftCutBasedId         = 1UL<<12,  
+      SoftMvaId              = 1UL<<13,  
+      MvaLoose               = 1UL<<14,  
+      MvaMedium              = 1UL<<15,  
+      MvaTight               = 1UL<<16,
+      MiniIsoLoose           = 1UL<<17,  // reliso<0.40
+      MiniIsoMedium          = 1UL<<18,  // reliso<0.20
+      MiniIsoTight           = 1UL<<19,  // reliso<0.10
+      MiniIsoVeryTight       = 1UL<<20   // reliso<0.05
+    };
+    
+    bool passed( unsigned int selection ) const { return (selectors_ & selection)==selection; }
```

Comment:
enums are known in ROOT as well.
It would be much easier to track the meaning of the call to "passed(id)" if it's done using the Selector enum as type.

Reply:
Here the choice of type is intentional to be able to do passed(TkIsoLoose|CutBasedIdHighPt), which would check that all flags are set. That's the most typical use case and it's a bit ugly and error prone to do it by hand on the bitmap itself.